src/main/java/com/bazaarvoice/jsonpps/PrettyPrintJson.java

```diff
@@ -308,4 +320,16 @@ private File getTemporaryFileFor(File file) {
         tempFile.deleteOnExit();
         return tempFile;
     }
+    
+    private PrettyPrinter getPrettyPrinter(String lf) {
+		DefaultPrettyPrinter prettyPrinter = new DefaultPrettyPrinter(lf);
+		if(tabs) {
+			prettyPrinter.indentArraysWith(new TabIndentation());
+			prettyPrinter.indentObjectsWith(new TabIndentation());
+		}
+		else {
+			prettyPrinter.indentArraysWith(DefaultIndenter.SYSTEM_LINEFEED_INSTANCE);
```

Comment:
Maybe the command-line argument can control whether we call indentArraysWith(...) regardless of whether we are using tabs.